wiki/mapping/how-to-testplay.md

```diff
@@ -35,7 +35,9 @@ water, written feedback instead of video is totally acceptable as you learn the
 
 :::warning
 It is important that you play the map in Practice Mode in order to prevent
-creating a new scoreboard on ScoreSaber if you have the mod installed!
+creating a new scoreboard on ScoreSaber if you have the mod installed! <br/>
+BeatLeader isn't effected so nothing to worry about
```

Comment:
Think should not be changed/mentioned as since it contradicts step 5 which is best practice that want to encourage.